src/commands/networks/removeNetwork.ts

```diff
@@ -13,11 +14,14 @@ export async function removeNetwork(context: IActionContext, node?: NetworkTreeI
     nodes = await multiSelectNodes(
         { ...context, suppressCreatePick: true, noItemFoundErrorMessage: 'No networks are available to remove' },
         ext.networksTree,
-        new RegExp(NetworkTreeItem.contextValue, 'i'),
+        NetworkTreeItem.customNetworkRegExp,
         node,
         nodes
     );
-
+    if (nodes.some(n => builtInNetworks.includes(n.networkName))) {
+        await ext.ui.showWarningMessage("The built-in networks 'bridge', 'host', and 'none' cannot be removed.");
```

Comment:
Probably shouldn't await this one as it will hold up until the user clicks X. You can remove the `await` and add `/* eslint-disable-next-line @typescript-eslint/no-floating-promises */` above that to prevent the linter error.